docs/en/operations/update.md

```diff
@@ -16,3 +16,15 @@ $ sudo service clickhouse-server restart
 If you installed ClickHouse using something other than the recommended `deb` packages, use the appropriate update method.
 
 ClickHouse does not support a distributed update. The operation should be performed consecutively on each separate server. Do not update all the servers on a cluster simultaneously, or the cluster will be unavailable for some time.
+
+The upgrade of older version of ClickHouse to specific version:
+
+As an example: 
+
+```bash
+$ sudo apt-get update
+$ sudo apt-get install clickhouse-server=20.12.4.5 clickhouse-client=20.12.4.5 clickhouse-common-static=20.12.4.5
```

Comment:
I think it's better not to specify the version like this, because it will quickly become obsolete and it will look like we don't update our documentation.

Reply:
My impression is, just give an example when people need to upgrade their current version to a specific version.  I can remove this version number and push it like this?
is it better?
sudo apt-get install clickhouse-server=xx.yy.a.b clickhouse-client=xx.yy.a.b clickhouse-common-static=xx.yy.a.b

Reply:
Maybe you can add something like this `where xx.yy.a.b is a current stable version. The latest stable version could be found [here](https://github.com/ClickHouse/ClickHouse/releases)`

Reply:
Okay! I will make it today!